### PR TITLE
11.0.0

### PR DESCRIPTION
**Pull request by uhafner**
Testing branch that contains all PRs in a single branch.

### CODE REVIEW

File: plugin/src/main/java/io/jenkins/plugins/analysis/core/steps/PublishIssuesStep.java [hunk 1]
```diff
@@ -884,20 +411,19 @@ protected ResultAction run() throws IOException, InterruptedException, IllegalSt
             }
             report.addAll(step.reports);
 
-            StageResultHandler statusHandler = new PipelineResultHandler(getRun(),
-                    getContext().get(FlowNode.class));
+            var deltaCalculator = DeltaCalculatorFactory
+                    .findDeltaCalculator(step.scm, getRun(), getWorkspace(), getTaskListener(), new FilteredLog());
```
Comment (jglick) [inline plugin/src/main/java/io/jenkins/plugins/analysis/core/steps/PublishIssuesStep.java:415]: Requiring a workspace here seems to be an incompatible change: a previously working pipeline that collected a list from multiple parallel agents using `scanForIssues` and finally published them together using `publishIssues` is now broken. Can the workspace be made optional?

Comment (uhafner) [reply plugin/src/main/java/io/jenkins/plugins/analysis/core/steps/PublishIssuesStep.java:415]: It looks like the GitClient accepts `null` as workspace. So it should be possible, can you please file an issue in Jira?

Comment (jglick) [reply plugin/src/main/java/io/jenkins/plugins/analysis/core/steps/PublishIssuesStep.java:415]: #1727

Comment (jglick) [reply plugin/src/main/java/io/jenkins/plugins/analysis/core/steps/PublishIssuesStep.java:415]: @uhafner can you take a look please? This is a critical regression in the current plugin release.

**Pull request merged**